returnn/torch/data/pipeline.py

```diff
@@ -542,13 +577,13 @@ def set_seed(self, seed: int) -> ShufflingDataPipe:
         Sets the seed for the next invocation of ``__iter__``, for compatibility with
         ``torch.utils.data.graph_settings.apply_random_seed``.
         """
-        self._seed = seed % (2**32)  # seed must be within [0, 2**32) for seeding RandomState
+        self._seed = abs(seed) % (2**32)  # seed must be within [0, 2**32) for seeding RandomState
```

Comment:
No need for the `abs`.
```suggestion
        self._seed = seed % (2**32)  # seed must be within [0, 2**32) for seeding RandomState
```